python-sdk/tests/benchmark/dags/evaluate_load_file.py

```diff
@@ -1,21 +1,26 @@
 import json
 import os
 from datetime import datetime
-from pathlib import Path
 
 from airflow import DAG
 from astro import sql as aql
 from astro.constants import DEFAULT_CHUNK_SIZE, FileType
 from astro.files import File
 from astro.sql.table import Metadata, Table
+from google.cloud import storage
+from tests.benchmark import settings as benchmark_settings
 
 START_DATE = datetime(2000, 1, 1)
 
 
 def load_config():
-    config_path = Path(Path(__file__).parent.parent, "config.json").resolve()
-    with open(config_path) as fp:
-        return json.load(fp)
+    tmp_config = "/tmp/config.json"
+    storage_client = storage.Client()
+    storage_client.download_blob_to_file(
+        file_obj=open(tmp_config, "wb"),
+        blob_or_uri=benchmark_settings.publish_benchmarks_config_path,
+    )
+    return json.load(open(tmp_config))
```

Comment:
Please use the `open` context manager. Otherwise in case of errors it might not close the file.

Reply:
It seems you are actually not interested about the file. In this case you could use [download_as_text](https://googleapis.dev/python/storage/latest/storage/blobs.html?highlight=download_as_string#google.cloud.storage.blob.Blob.download_as_text) and just pass the result to `json.loads`.